src/containers/ReadEventPage.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { connect } from "react-redux";
+import moment from "moment";
+import PropTypes from "prop-types";
+
+import { removeEvent } from "../actions/events";
+import { historyPropTypes } from "../common/models";
+
+class ReadEventPage extends React.Component {
+  onEditClick = () => {
+    this.props.history.push(`/edit/${this.props.event.id}`);
+  };
+
+  onRemoveClick = () => {
+    this.props.removeEvent(this.props.event.id);
+    this.props.history.push("/");
+  };
+
+  render() {
+    const {
+      image,
+      title,
+      startDate,
+      endDate,
+      organisator,
+      city,
+      category,
+      description,
+    } = this.props.event;
+
+    const start = moment(startDate).format("D MMM");
+    const end = moment(endDate).format("D MMM");
+
+    return (
+      <>
+        {image ? (
```

Comment:
Hm, isn't this component exactly the same as I saw before? In EditPage or something? If so, then it would be better to split common part as a separate component and reuse it :)

Reply:
There is a slight difference, but I was considering creation of a new component for that ;)